compiler/.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
shouldn't this file be ignored by `.gitignore`?

Reply:
Yes, this file should be dropped from git

Reply:
But now it allows to run and debug Compiler-module using executable file with some kind of use-cases. So, I'm confused a little. Could You help me with this issue?